schemas/account.js

```diff
@@ -0,0 +1,20 @@
+const email = {
+  type: 'string'
```

Comment:
We have stronger schema here: https://github.com/digitalbazaar/bedrock-validation/blob/master/schemas/email.js

`schemas` are exported in bedrock-validation here: https://github.com/digitalbazaar/bedrock-validation/blob/master/lib/index.js#L21

They are implemented as schema factories, so you do `schema.email()`.